domains/measurements/Measurements.API/Tests/Models/MeteringPointTests.cs

```diff
@@ -0,0 +1,60 @@
+using Xunit;
+using System;
+using API.MeteringPoints.Api.v2024_01_10.Dto.Responses;
+using API.MeteringPoints.Api.v2024_01_10.Dto.Responses.Enums;
+using FluentAssertions;
+
+namespace Tests.Models
+{
+    public class MeteringPointTests
+    {
+        [Theory]
+        [InlineData("E17", MeterType.consumption)]
+        [InlineData("E18", MeterType.production)]
+        [InlineData("D01", MeterType.child)]
+        [InlineData("D99", MeterType.child)]
+        public void MeteringPoints_MeterType_Success(string stringType, MeterType targetType)
+        {
+            var mt = MeteringPoint.GetMeterType(stringType);
+
+            Assert.Equal(targetType, mt);
+        }
+
+        [Fact]
+        public void MeteringPoints_MeterTypeChildSeries_Success()
```

Comment:
Can this test be more descriptive in its naming?

Reply:
done